### PR TITLE
Fix: about us page

### LINKED ISSUE #171: Empty about-us page
## Describe the bug
On the Homepage continue learning section I can see a horizontal scrollbar on cards
I can't see anything on the about us page and can't render faqs pages
there is a console.log on the privacy page

## To Reproduce

1. You can see the scrollbar on the homepage
2. try to go to /about 
3. try to go to /faqs
4. see in the console after going to /privacy

## Expected behavior

- There shouldn't be a scrollbar
- All pages should render properly
- There shouldn't any console.log  

## Environment (please complete the following information):

- OS: Linux: Ubuntu
- Browser: Firefox

### PR DESCRIPTION
**Pull request by rajatgupta24**
**Issue Number**

fixes #171 

**Describe the changes you've made**

Removed the scrollbar from continue cards.
Fix faq, about us page.
Removed the console.log

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**
NA

**Additional context (OPTIONAL)**

**Test plan (OPTIONAL)**

A good test plan should give instructions that someone else can easily follow.

**Checklist**

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: src/pages/Info/About/index.js [hunk 1]
```diff
@@ -1,7 +1,93 @@
...
+      <Typography className={classes.subHeading} variant="h2" color="textPrimary" component="h1">
+        Our Vision
+      </Typography>
+      <Typography className={classes.content}>
+        Nulla facilisi. Phasellus sollicitudin nulla et quam mattis feugiat. Aliquam eget maximus
+        est, id dignissim quam. Nulla facilisi. Phasellus sollicitudin nulla et quam mattis feugiat.
+        Aliquam eget maximus est, id dignissim quam. Nulla facilisi. Phasellus sollicitudin nulla et
+        quam mattis feugiat. Aliquam eget maximus est, id dignissim quam.
+      </Typography>
+      <Typography className={classes.subHeading} variant="h2" color="textPrimary" component="h1">
```
Comment (Abhishek-kumar09) [inline src/pages/Info/About/index.js:74]: These things will go in docs layout, read how to use mdx.
Please put it in docs layout just like `Privacy` page.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

File: src/pages/Info/About/index.js (hunk 1 above)
Comment (Abhishek-kumar09) [reply src/pages/Info/About/index.js:74]: We can add BottomNavbar in docslayout

Comment (rajatgupta24) [reply src/pages/Info/About/index.js:74]: Ok

**Comment by Abhijay007** [comment]
@rajatgupta24 there are some conflicts in the `routes.js` file although the code looks fine. You may need to pull upstream changes once and push them in order to resolve these conflicts. The `routes.js` file is updated with recent changes.

**Comment by rajatgupta24** [comment]
> @rajatgupta24 there are some conflicts in the `routes.js` file although the code looks fine. You may need to pull upstream changes once and push them in order to resolve these conflicts. The `routes.js` file is updated with recent changes.

Sorry for the delay, I wasn't well :face_with_thermometer:

**Comment by Abhijay007** [comment]
> Sorry for the delay, I wasn't well 🤒

it's ok buddy take care !!

**Review by Abhijay007 — COMMENTED** [review]
@rajatgupta24  
LGTM 👍

**Comment by Abhishek-kumar09** [comment]
Github, rebase

**Pull request merged**